Dockerfile

```diff
@@ -60,21 +60,24 @@ RUN useradd --create-home --shell /bin/bash critiquebrainz
 
 # Consul Template service is already set up with the base image.
 # Just need to copy the configuration.
-COPY ./docker/prod/consul-template.conf /etc/consul-template.conf
+COPY ./docker/consul-template.conf /etc/consul-template.conf
```

Comment:
We don't need this comment any more, it's related to the old version. The file doesn't need to be copied to this location anymore either. We should copy it to `/etc/consul-template-uwsgi.conf` for uwsgi, and a separate file `/etc/consul-template-cron.conf` for cron. The cron job needs no `exec{}` block. We just use it to render the config file in the cron container.